mig/shared/functionality/cat.py

```diff
@@ -153,6 +173,12 @@ def main(client_id, user_arguments_dict, environ=None):
                                    'name': pattern})
             status = returnvalues.FILE_NOT_FOUND
 
+        if not _check_sizes(configuration, paths=match):
+            status = returnvalues.REJECTED_ERROR
+            output_objects.append({'object_type': 'error_text', 'text':
+                                       "request exceeded size limit"})
```

Comment:
I'd really like to extend this error message to point to the site configuration limit and the user workarounds e.g. with something like:
"Site configuration prevents web serving contents bigger than %(limit)d bytes - please use alternatives like %(enabled_protos)s to serve these contents"
with the conf limit value and enabled sftp/ftps/webdavs proto names merged in.

Reply:
https://github.com/ucphhpc/migrid-sync/pull/131/files#diff-bfd4061d9bc77336a6c4d2608e4b94357d5d4e295b4d9fc5a29c3cb295e85b15R140